Tests/Tests/STPColorUtilsTest.m

```diff
@@ -103,38 +103,42 @@ - (void)testAllColorSpaces {
         }
     };
 
-    CFStringRef colorSpaceNames[] = {
-      kCGColorSpaceSRGB,
-      kCGColorSpaceDCIP3,
-      kCGColorSpaceROMMRGB,
-      kCGColorSpaceITUR_709,
-      kCGColorSpaceDisplayP3,
-      kCGColorSpaceITUR_2020,
-      kCGColorSpaceGenericRGB,
-      kCGColorSpaceGenericXYZ,
-      kCGColorSpaceLinearGray,
-      kCGColorSpaceLinearSRGB,
-      kCGColorSpaceGenericCMYK,
-      kCGColorSpaceGenericGray,
-      kCGColorSpaceACESCGLinear,
-      kCGColorSpaceAdobeRGB1998,
-      kCGColorSpaceExtendedGray,
-      kCGColorSpaceExtendedSRGB,
-      kCGColorSpaceGenericRGBLinear,
-      kCGColorSpaceExtendedLinearGray,
-      kCGColorSpaceExtendedLinearSRGB,
-      kCGColorSpaceGenericGrayGamma2_2,
-    };
+    NSArray *colorSpaceNames = @[
+      (__bridge NSString *)kCGColorSpaceSRGB,
+      (__bridge NSString *)kCGColorSpaceDCIP3,
+      (__bridge NSString *)kCGColorSpaceROMMRGB,
+      (__bridge NSString *)kCGColorSpaceITUR_709,
+      (__bridge NSString *)kCGColorSpaceDisplayP3,
+      (__bridge NSString *)kCGColorSpaceITUR_2020,
+      (__bridge NSString *)kCGColorSpaceGenericRGB,
+      (__bridge NSString *)kCGColorSpaceGenericXYZ,
+      (__bridge NSString *)kCGColorSpaceLinearSRGB,
+      (__bridge NSString *)kCGColorSpaceGenericCMYK,
+      (__bridge NSString *)kCGColorSpaceGenericGray,
+      (__bridge NSString *)kCGColorSpaceACESCGLinear,
+      (__bridge NSString *)kCGColorSpaceAdobeRGB1998,
+      (__bridge NSString *)kCGColorSpaceExtendedGray,
+      (__bridge NSString *)kCGColorSpaceExtendedSRGB,
+      (__bridge NSString *)kCGColorSpaceGenericRGBLinear,
+      (__bridge NSString *)kCGColorSpaceExtendedLinearSRGB,
+      (__bridge NSString *)kCGColorSpaceGenericGrayGamma2_2,
+      ];
+
+    if (@available(iOS 10.0, *)) {
+        colorSpaceNames = [colorSpaceNames arrayByAddingObjectsFromArray:@[
+                                                                           (__bridge NSString *)kCGColorSpaceLinearGray,
```

Comment:
Instead of adding all this (ugly?) bridging, since it was only two iOS 10 cases, did you consider just following the pattern that the iOS 11-only case `kCGColorSpaceGenericLab` took?

Reply:
Oh yeah, I was so focused on struggling with this Core Foundation syntax I forgot to try that.